Homework-05-files-encoding-formats.ipynb

```diff
@@ -46,11 +101,104 @@
+      "Title:___ RDF 1.1 Concepts and Abstract Syntax\n",
+      "Authors:___ [{'name': ' W3C', 'uri': None, 'localAuthorityId': None, 'firstName': '', 'lastName': 'W3C', 'positionInList': 0, 'affiliations': None}]\n",
+      "itemType:___ WEBPAGE\n",
+      "Title:___ Representing Contextualized Data using Semantic Web Tools\n",
+      "Authors:___ [{'name': 'Robert Macgregor', 'uri': None, 'localAuthorityId': None, 'firstName': 'Robert', 'lastName': 'Macgregor', 'positionInList': 0, 'affiliations': None}, {'name': 'In-Young Ko', 'uri': None, 'localAuthorityId': None, 'firstName': 'In-Young', 'lastName': 'Ko', 'positionInList': 1, 'affiliations': None}]\n",
+      "itemType:___ CONFERENCE_PAPER\n",
+      "Title:___ Term Extraction and Automatic Indexing\n",
+      "Authors:___ [{'name': 'Christian Jacquemin', 'uri': None, 'localAuthorityId': None, 'firstName': 'Christian', 'lastName': 'Jacquemin', 'positionInList': 0, 'affiliations': None}, {'name': 'Didier Bourigault', 'uri': None, 'localAuthorityId': None, 'firstName': 'Didier', 'lastName': 'Bourigault', 'positionInList': 1, 'affiliations': None}, {'name': 'Ruslan Mitkov', 'uri': None, 'localAuthorityId': None, 'firstName': 'Ruslan', 'lastName': 'Mitkov', 'positionInList': 2, 'affiliations': None}]\n",
+      "itemType:___ BOOK_SECTION\n",
+      "Title:___ The Anatomy of a Nanopublication\n",
+      "Authors:___ [{'name': 'Paul Groth', 'uri': None, 'localAuthorityId': None, 'firstName': 'Paul', 'lastName': 'Groth', 'positionInList': 0, 'affiliations': None}, {'name': 'Andrew Gibson', 'uri': None, 'localAuthorityId': None, 'firstName': 'Andrew', 'lastName': 'Gibson', 'positionInList': 1, 'affiliations': None}, {'name': 'Jan Velterop', 'uri': None, 'localAuthorityId': None, 'firstName': 'Jan', 'lastName': 'Velterop', 'positionInList': 2, 'affiliations': None}]\n",
+      "itemType:___ JOURNAL_ARTICLE\n",
+      "Title:___ rrrr\n",
+      "Authors:___ ffff\n",
+      "itemType:___ asdawe\n",
+      "Title:___ aaa\n",
+      "Authors:___ ddd\n",
+      "itemType:___ ccc\n",
+      "Title:___ fff\n",
+      "Authors:___ iii\n",
+      "itemType:___ yyy\n",
+      "Title:___ ttttt\n",
+      "Authors:___ 777777777yyyyyyyy\n",
+      "itemType:___ fffffff\n",
+      "Title:___ xxxxx\n",
+      "Authors:___ sssssss\n",
+      "itemType:___ ddddddddd\n",
+      "Title:___ eeeeee\n",
+      "Authors:___ rrrrr\n",
+      "itemType:___ ttttt\n",
+      "Press 0 to add a new entry Press 1 to exit0\n",
+      "What is the title you want to type in for your entry?ffffffffffddddddddddd\n",
+      "What is the author you want to type in for your entry?ssssssssss\n",
+      "What is the item type you want to type in for your entry?dddddddddddddd\n",
+      "Press 0 to add a new entry Press 1 to exit1\n"
+     ]
+    }
+   ],
    "source": [
-    "# your code"
+    "import json\n",
+    "\n",
+    "#C:\\\\Users\\\\biogx\\\\GXH\\\\HW-05-data.json\n",
+    "\n",
+    "def ReadJson(path):\n",
```

Comment:
functions in python use snake case (lowercase with "_" separating words, e.g. `snake_case`)

Reply:
other than that, good use of functions! if you return `data` from your function, you don't have to load it again.